### ISSUE TITLE
amass reverse whois produce panic: runtime error: invalid memory address or nil pointer dereference

### ISSUE THREAD

**Issue by idanshimon** [OWASP/Amass#276]
I am using amass 3.1.8 and got a panic when running the following command:
```bash
$ amass --version
v3.1.8
```
```bash
$ amass intel -whois -d uber.com
```
```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x8 pc=0xa61396]

goroutine 13 [running]:
github.com/OWASP/Amass/resolvers.(*ResolverPool).numUsableResolvers(0x0, 0xc000932360)
	/Users/caffix/go/src/github.com/OWASP/Amass/resolvers/pool.go:413 +0x26
github.com/OWASP/Amass/resolvers.(*ResolverPool).Resolve(0x0, 0x1049fba, 0xe, 0x103be78, 0x1, 0x1, 0x458bc0, 0xc001717db8, 0xc001717dc0, 0x8, ...)
	/Users/caffix/go/src/github.com/OWASP/Amass/resolvers/pool.go:226 +0x40
github.com/OWASP/Amass/services/sources.(*RADb).OnStart(0xc00097ae00, 0x14de248, 0xc00097ae00)
	/Users/caffix/go/src/github.com/OWASP/Amass/services/sources/radb.go:60 +0x69
github.com/OWASP/Amass/services.(*BaseService).Start(0xc00097ae00, 0x0, 0x0)
	/Users/caffix/go/src/github.com/OWASP/Amass/services/service.go:173 +0x17e
github.com/OWASP/Amass/intel.(*Collection).ReverseWhois(0xc000922720, 0x0, 0x0)
	/Users/caffix/go/src/github.com/OWASP/Amass/intel/intel.go:360 +0x21b
created by main.runIntelCommand
	/Users/caffix/go/src/github.com/OWASP/Amass/cmd/amass/intel.go:216 +0xc9b
```

**Comment by i223t** [comment]
The same:

docker run -v ~/amass:/amass/ amass intel -whois -d example.com
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x8 pc=0xa674f6]

goroutine 26 [running]:
github.com/OWASP/Amass/resolvers.(*ResolverPool).numUsableResolvers(0x0, 0xc00000e340)
	/go/src/github.com/OWASP/Amass/resolvers/pool.go:413 +0x26
github.com/OWASP/Amass/resolvers.(*ResolverPool).Resolve(0x0, 0x1052905, 0xe, 0x10447d8, 0x1, 0x1, 0x45a970, 0xc00168ddb8, 0xc00168ddc0, 0x8, ...)
	/go/src/github.com/OWASP/Amass/resolvers/pool.go:226 +0x40
github.com/OWASP/Amass/services/sources.(*RADb).OnStart(0xc00001ee00, 0x14e6c58, 0xc00001ee00)
	/go/src/github.com/OWASP/Amass/services/sources/radb.go:60 +0x69
github.com/OWASP/Amass/services.(*BaseService).Start(0xc00001ee00, 0x0, 0x0)
	/go/src/github.com/OWASP/Amass/services/service.go:173 +0x17e
github.com/OWASP/Amass/intel.(*Collection).ReverseWhois(0xc00092e780, 0x0, 0x0)
	/go/src/github.com/OWASP/Amass/intel/intel.go:360 +0x21b
created by main.runIntelCommand
	/go/src/github.com/OWASP/Amass/cmd/amass/intel.go:216 +0xc9b

**Comment by caffix** [comment]
Thank you for the feedback. The fix will be in release v3.1.9

**Comment by nicolastrognot** [comment]
When will it be released ?
